src/RequestLogger/RequestLogger.csproj

```diff
@@ -8,11 +8,20 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="MappingGenerator" Version="2023.3.56" />
```

Comment:
Unfamiliar with `MappingGenerator` - what is it?

Reply:
sorry, must have pressed something to add it! It's really a plugin as opposed to package.  It's equivalent to automapper, but actually generates the mapping code and I used it for the request converters.

How it works is that you write a method like this:

```
public static HttpRequest ConvertRequest(Request request)
{
}
```

then you open the context menu and select `generate mapping code`, and it generates this:

```
public static HttpRequest ConvertRequest(Request request)
    {
        return new HttpRequest
        {
            Verb = request.Verb,
            Service = request.Service,
            Customer = request.Customer,
            Path = request.Path,
            QueryParams = request.QueryParams?.ToString(),
            Body = request.Body,
            Headers = request.Headers.ToString(),
            RequestTime = request.RequestTime
        };
    }
```

granted I could have written that myself easily, but it's really helpful for large conversions